cupyx/time.py

```diff
@@ -33,14 +33,17 @@ def _to_str_per_item(t):
 
     def to_str(self, show_gpu=False):
         ts = self._ts if show_gpu else self._ts[[0]]
+        devices = ['CPU', 'GPU']
         return '{:<20s}:{}'.format(
-            self.name, ' '.join([self._to_str_per_item(t) for t in ts]))
+            self.name, ' '.join(["    " + devices[i] + ':'
+                                 + self._to_str_per_item(ts[i])
+                                 for i in range(len(ts))]))
```

Comment:
How about adding `device_name` parameter to `_to_str_per_item`?

Reply:
Done.

Reply:
tests are also passing.

Reply:
How about writing as follows?
```py
return '{:<20s}:{}'.format(
    self.name,
    ' '.join([self._to_str_per_item(*p) for p in zip(ts, device_names)])
```

Reply:
Done

Reply:
Why did you change the indentation?

Reply:
It's changed by `autopep8`

Reply:
Because the length of the last line becomes (80>79) there.

Reply:
Please merge [this PR](https://github.com/rushabh-v/cupy/pull/1) for readability.

Reply:
Merged!
Thanks. That looks much clearer now😄